content/2015.09.kinto-encryption-example.rst

```diff
@@ -0,0 +1,207 @@
+How to encrypt data with Kinto.js
+###########################################
+
+:date: 2015-09-16
+:summary: How to encrypt data with Kinto.js.
+:slug: kinto-encryption-example
+:lang: en
+:url: kinto-encryption-example
+
+A guest post this time, by Michiel de Jong from the Firefox OS team! Sorry for
+writing in English. :)
+
+If you want to encrypt data that is uploaded to the cloud by an offline-first app,
+it makes sense to keep the local copy of the data in-the-clear, and encrypt the
+data only just before it is uploaded. That way, you can easily and efficiently display
+and update the local copy of the data, without the repeated cost of encrypting
+and decrypting everything all the time. Likewise, you only need to decrypt incoming data
+once, when it's downloaded, and your app can then display the data as often as
+needed without extra processing costs.
+
+Kinto.js now supports
```

Comment:
I realize we're introducing Kinto.js without having never explained its purpose before on this blog… It may be worth adding a few lines for presenting its purpose, mentioning that a blog post will come next getting into more details about it.